src/Compilers/VisualBasic/Portable/Binding/MemberSemanticModel.vb

```diff
@@ -793,17 +793,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return GetSymbolInfoForNode(options, GetBoundNodeSummary(node), binderOpt:=Nothing)
         End Function
 
-        Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, options As GetOperationOptions, cancellationToken As CancellationToken) As IOperation
-            Dim summary = GetBoundNodeSummary(node)
-            Dim result As BoundNode
-            Select Case options
-                Case GetOperationOptions.Highest
-                    result = summary.HighestBoundNode
-                Case GetOperationOptions.Parent
-                    result = summary.LowestBoundNodeOfSyntacticParent
-                Case Else
-                    result = summary.LowestBoundNode
-            End Select
+        Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, cancellationToken As CancellationToken) As IOperation
+            ' see whether we can bind smaller scope than GetBindingRoot to make perf better
```

Comment:
>see whether we can bind smaller scope than GetBindingRoot to make perf better [](start = 14, length = 77)

Please add a link to the issue tracking this work.

Reply:
done